configure.ac

```diff
@@ -1,7 +1,7 @@
 # Process this file with autoconf to produce a configure script.
 
 AC_PREREQ([2.69])
-AC_INIT([libdfp],[1.0.12],[tulioqm@br.ibm.com])
+AC_INIT([libdfp],[1.0.12],[murphyp@linux.vnet.ibm.com])
```

Comment:
The AC_INIT documentation states "The optional argument bug-report should be the email to which users should send bug reports." Is there any precedent for using something other than an email?